src/components/editTestCase/qdm/LeftPanel/LeftPanelNavTabs.tsx

```diff
@@ -25,14 +34,16 @@ export default function RightPanelNavTabs(props: NavTabProps) {
         data-testid="elements-tab"
         value="elements"
       />
-      <Tab
-        tabIndex={0}
-        aria-label="JSON tab panel"
-        type="D"
-        data-testid="json-tab"
-        label="JSON"
-        value="json"
-      />
+      {!qdmHidJsonTab && (
```

Comment:
can we not do this? `!featureFlags?.qdmHideJson` directly?

Reply:
@adongare This is done, thanks a lot Ashok!